apps/web/src/services/administration/iam/user/modules/user-management-modal/modules/UserInfoForm.vue

```diff
@@ -0,0 +1,280 @@
+                       class="input-form"
+        >
+            <p-text-input v-model="formState.name"
+                          class="text-input"
+                          autocomplete="username"
+                          @update:value="handleChangeInput"
+            />
+        </p-field-group>
+    </div>
+</template>
+
+<script setup lang="ts">
+import { computed, reactive, watch } from 'vue';
+import type { TranslateResult } from 'vue-i18n';
+
+import {
+    PFieldGroup, PTextInput, PButton, PFilterableDropdown,
+} from '@spaceone/design-system';
+import type { MenuItem } from '@spaceone/design-system/types/inputs/context-menu/type';
+import { debounce } from 'lodash';
+
+
+import { SpaceConnector } from '@cloudforet/core-lib/space-connector';
+
+import { store } from '@/store';
+import { i18n } from '@/translations';
+
+import type { UserReferenceMap } from '@/store/modules/reference/user/type';
+
+import ErrorHandler from '@/common/composables/error/errorHandler';
+
+import type {
+    Validation,
+} from '@/services/administration/iam/user/lib/user-form-validations';
+import {
+    checkDuplicateID,
+    checkEmailFormat, checkEmptyValue,
+    checkOauth,
+    checkRequiredField,
+} from '@/services/administration/iam/user/lib/user-form-validations';
+import type { User } from '@/services/administration/iam/user/type';
+import { useUserPageStore } from '@/services/administration/store/user-page-store';
+
+interface Props {
+    activeTab?: string;
+    item?: User;
+}
+const props = withDefaults(defineProps<Props>(), {
+    activeTab: '',
+    item: undefined,
+});
+
+const userPageStore = useUserPageStore();
+const userPageState = userPageStore.$state;
+
+const emit = defineEmits<{(e: 'change-input', formState): void}>();
+
+const state = reactive({
+    loading: false,
+    searchText: '',
+    selectedItems: [] as MenuItem[],
+    externalItems: [] as MenuItem[],
+    supportFind: computed(() => !!store.state.domain.authOptions?.support_find),
+    users: computed<UserReferenceMap>(() => store.getters['reference/userItems']),
+});
+const formState = reactive({
+    userId: '',
+    name: '',
+});
+const validationState = reactive({
+    isUserIdValid: undefined as undefined | boolean,
+    userIdInvalidText: '' as TranslateResult | string,
+    userIdValidText: '' as TranslateResult | string,
+});
+
+/* Components */
+const setExternalMenuItems = (users) => {
+    state.externalItems = [];
+    users.forEach((user) => {
+        const singleItem = {
+            name: user.userId,
+            label: user.name ? `${user.userId} (${user.name})` : user.userId,
+            disabled: false,
+        };
+        if (state.users[user.userId]) {
+            singleItem.label = `(${i18n.t('IDENTITY.USER.FORM.ALREADY_EXISTS')}) ${singleItem.label}`;
+            singleItem.disabled = true;
+        }
+        state.externalItems.push(singleItem);
+    });
+};
+const onDeleteSelectedExternalUser = () => {
```

Comment:
nitpick: Please use `handle-` prefix for event function.

```suggestion
const handleDeleteSelectedExternalUser = () => {
```